airflow/providers/google/cloud/hooks/bigquery.py

```diff
@@ -70,6 +74,29 @@ class BigQueryHook(GoogleBaseHook, DbApiHook):
     conn_name_attr = 'gcp_conn_id'
     default_conn_name = 'google_cloud_default'
     conn_type = 'google_cloud_platform'
+    hook_name = 'Google Cloud'
+
+    @staticmethod
+    def monkey_patch_connection_form(form_to_patch):
```

Comment:
Shouldn't that be in hooks.base_google?

Reply:
Yep. Moved. it there and added a code that only adds connection form widgets from the base classes, not from the derived ones.